ruby/README.md

```diff
@@ -0,0 +1,15 @@
+# Github-Repo-Cloner (Ruby)
+
+Clone any user's public repositories concurrently.
+
+### Requirements
+ - Ruby 3.2.1 (or greater)
+
+### Local Setup
+1. Clone main project: `git clone git@github.com:murjax/Github-Repo-Cloner.git`
+2. Navigate to ruby project: `cd ruby`
+3. Install dependencies: `bundle install`
+
+### Usage
+1. Run existing binary: `./bin/github_repo_cloner <username>`
+2. Run tests: `bundle exec rspec`
```

Comment:
can we put the usage section above the local setup section?

I think the local setup section is like a how to contribute section.
I dont think you need to do bundle install to run the binary?

also should run the tests be in the local setup section?